javascript/javascript1/week2/index2.js

```diff
@@ -0,0 +1,26 @@
+
+
+function getFullname (fullname1 ,fullname2,useFormalName,gender){
+    if(fullname1!="" && fullname2!="")
+    {       
+        if(gender==="male"){
+            if (useFormalName ===true)
+            return("Lord"+" "+fullname1+" "+fullname2)
+         else
+            return (fullname1+" "+fullname2)
+        } else if(gender==="female"){
+            if (useFormalName ===false)
+            return("Ms"+" "+fullname1+" "+fullname2)
+         else
+            return (fullname1+" "+fullname2)
+        }
+        
+    }else 
+       return("you need to write youe name and your family")       
+}
+
+
+console.log(getFullname("","Hughes",true,"male"))
+console.log(getFullname("Benjamin","Hughes",true,"male"))
+console.log(getFullname("Benjamin","Hughes",false,"male"))
+console.log(getFullname("Bahareh","Sichani",false,"female"))
```

Comment:
Good solution! 👍

Reply:
i think you should use || in stead of && in line 4.

Reply:
@chunmei the vars fullname1 and fullname2 respond to name and surname respectively, so you need to use && to return the full name. Or do you see something that I'm missing? 🤔